tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -42,7 +42,7 @@ sub reboot_and_wait_up {
         #login
         #The timeout can't be too small since autoyast installation
         #need to wait 2nd phase install to finish
-        assert_screen("text-login", 600);
+        assert_screen("tty1-selected", 600);
```

Comment:
this change make all our tests fail like this:
https://openqa.nue.suse.com/tests/3853054

i added a new needle to get around this as the change is still under discussion in the progress ticked.

Reply:
will add ttys1-selected to cover this issue correctly, thanks